src/l500/l500-depth.cpp

```diff
@@ -354,6 +355,44 @@ namespace librealsense
         AC_LOG( INFO, "Depth sensor calibration has been reset" );
     }
 
+    float l500_depth_sensor::get_max_usable_range() const
+    {
+        if( !supports_option( RS2_OPTION_MAX_USABLE_RANGE ) )
+            throw librealsense::wrong_api_call_sequence_exception( "max usable range option is not supported" );
+
+        if( get_option( RS2_OPTION_MAX_USABLE_RANGE ).query() != 1.0f )
+            throw librealsense::wrong_api_call_sequence_exception( "max usable range option is not on" );
+
+        if( ! is_streaming() )
+        {
+            throw librealsense::wrong_api_call_sequence_exception("depth sensor is not streaming!");
+        }
+
+       algo::max_range::max_usable_range mur;
```

Comment:
Not sure if I prefer another usage paradigm... let's talk about?

Reply:
as discussed,
Changed to function instead of class under this namespace
librealsense::algo::max_usable_range::l500